operations/node_management/Adding_a_Liquid-cooled_blade_to_a_System_Using_SAT.md

```diff
@@ -0,0 +1,250 @@
+# Adding a Liquid-cooled blade to a System Using SAT
+
+This procedure will add a liquid-cooled blade to an HPE Cray EX system.
+
+## Prerequisites
+
+- The Cray command line interface \(CLI\) tool is initialized and configured on the system.
+
+- Knowledge of whether DVS is operating over the Node Management Network (NMN) or the High Speed Network (HSN).
+
+- Blade is being added to an existing liquid-cooled cabinet in the system.
+
+- The Slingshot fabric must be configured with the desired topology for desired state of the blades in the system.
+
+- The System Layout Service (SLS) must have the desired HSN configuration.
+
+- Check the status of the high-speed network (HSN) and record link status before the procedure.
+
+- The System Admin Toolkit \(SAT\) is installed and configured on the system.
+
+- The blades must have the coolant drained and filled during the swap to minimize cross-contamination of cooling systems.
+  - Review procedures in *HPE Cray EX Coolant Service Procedures H-6199*
+  - Review the *HPE Cray EX Hand Pump User Guide H-6200*
+
+### Use SAT to add the blade to hardware management
+
+1. Use the `sat swap` command to map the nodes' ethernet interface MAC addresses to the appropriate IP addresses and component xnames, and begin discovery for the blade.
+
+   The `--src-mapping` and `--dst-mapping` arguments may be used to pass in the ethernet interface mapping files containing the IP
+   addresses, MAC addresses, and component xnames for the nodes on the blade. If the slot was previously populated, the file passed into the
+   `--src-mapping` argument should be the mapping file saved during the [Removing a Liquid-cooled Blade from a System Using
+   SAT](Removing_a_Liquid-cooled_blade_from_a_System_Using_SAT.md) procedure, and the `--dst-mapping` argument should be a mappings file of
+   the MAC addresses, IP addresses, and component xnames from the new blade. If the new blade was removed from another system, the mappings
+   file was saved while performing the [Removing a Liquid-cooled Blade from a System Using
+   SAT](Removing_a_Liquid-cooled_blade_from_a_System_Using_SAT.md) procedure on the other system.
+
+   ```bash
+   ncn# sat swap blade --src-mapping <SRC_MAPPING> --dst-mapping <DST_MAPPING> -a enable <SLOT_XNAME>
+   ```
+
+   If the slot was not previously populated, the `--src-mapping` and `--dst-mapping` arguments should be omitted.
+
+   ```bash
+   ncn# sat swap blade -a enable <SLOT_XNAME>
+   ```
+
+### Power on and boot the nodes
+
+1. Use `sat bootsys` to power on and boot the nodes. Specify the appropriate BOS template for the node type.
+
+   ```bash
+   ncn# BOS_TEMPLATE=cos-2.0.30-slurm-healthy-compute
+   ncn# sat bootsys boot --stage bos-operations --bos-limit <SLOT_XNAME> --recursive --bos-templates $BOS_TEMPLATE
+   ```
+
+#### Check firmware
+
+1. Verify that the correct firmware versions for node BIOS, node controller (nC), NIC mezzanine card (NMC), GPUs, and so on.
```

Comment:
Provide a link in this step to the FAS documentation on how to check these versions.